util/dockerfiles/docker-bake.hcl

```diff
@@ -81,3 +103,88 @@ target "ubuntu-20-04_all-dependencies" {
   context = "ubuntu-20.04_all-dependencies"
   tags = ["${IMAGE_URI}/ubuntu-20.04_all-dependencies:${TAG}"]
 }
+
+target "ubuntu-22-04_min-dependencies" {
+  inherits = ["common"]
+  dockerfile = "Dockerfile"
+  context = "ubuntu-22.04_min-dependencies"
+  tags = ["${IMAGE_URI}/ubuntu-22.04_min-dependencies:${TAG}"]
+}
+
+target "gcc-compilers-a" {
```

Comment:
I understand why you did it this way. Perhaps we can call these `gcc-compilers-base-20-04` and `gcc-compilers-base-22-04`. Likewise for clang? It's just a bit more descriptive.

Reply:
I've made an additional commit with the requested change. If it's fine for it to be a separate commit, then I believe everything is ready to proceed.